### PR TITLE
Add keycloak probes config

### PR DESCRIPTION
**Pull request by isindir**
* Make Liveness and Readiness Probes initialDelay and Timeout configurable in incubator/keycloak charm
* Bump version of keycloak and chart

### CODE REVIEW

File: incubator/keycloak/templates/statefulset.yaml [hunk 1]
```diff
@@ -84,14 +84,14 @@ spec:
             httpGet:
               path: /auth/
               port: http
-            initialDelaySeconds: 120
-            timeoutSeconds: 5
+            initialDelaySeconds: {{ default 120 .Values.keycloak.livenessInitialDelaySeconds }}
```
Comment (unguiculus) [inline incubator/keycloak/templates/statefulset.yaml]: Remove default function calls. They are redundant. Defaults should be in `values.yaml`.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by isindir** [comment]
/cc @unguiculus 
Hi Reinhard, please see changes, thanks

**Comment by monotek** [comment]
Maybe the values should be higher by default? 
In GKE keycloak is restarted constantly because it starts way to slow.

File: incubator/keycloak/values.yaml [hunk 2]
```diff
@@ -43,6 +43,11 @@ keycloak:
   nodeSelector: {}
   tolerations: []
 
+  livenessInitialDelaySeconds: 120
+  livenessTimeoutSeconds: 5
+  readinessInitialDelaySeconds: 30
+  readinessTimeoutSeconds: 1
+
```
Comment (arroo) [inline incubator/keycloak/values.yaml]: would about putting these under a `probes` object?

**Comment by isindir** [comment]
@monotek Agree values are way to low, can change defaults here if there is enough interest...

File: incubator/keycloak/values.yaml (hunk 2 above)
Comment (isindir) [reply incubator/keycloak/values.yaml]: Not sure, these are just few self explanatory values..

Comment (unguiculus) [reply incubator/keycloak/values.yaml]: I'd suggest you do this:

```yaml
livenessProbe:
  initialDelaySeconds: 120
  timeoutSeconds: 5
readinessProbe:
  initialDelaySeconds: 30
  timeoutSeconds: 1
```

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by isindir** [comment]
updated, please review again

**Review by unguiculus — APPROVED** [review]

**Pull request merged**